system/lib/split_malloc.cpp

```diff
@@ -71,7 +71,7 @@ struct Space {
       start = split_memory*index;
     } else {
       // small area in existing chunk 0
-      start = EM_ASM_INT_V({ return (DYNAMICTOP+3)&-4; });
+      start = EM_ASM_INT_V({ return (HEAPU32[DYNAMICTOP_PTR>>2]+3)&-4; });
```

Comment:
Ended up bumping the version for both - that'll make regression testing a bit easier in case that will be necessary for someone (let's hope not).